src/.clang-format

```diff
@@ -15,7 +15,7 @@ BreakBeforeBinaryOperators: false
 BreakBeforeBraces: Allman
 BreakBeforeTernaryOperators: false
 BreakConstructorInitializersBeforeComma: false
-ColumnLimit:     0
+ColumnLimit:     120
```

Comment:
I would suggest to put the limit at 80, but put rather low value for PenaltyExcessCharacter and a high one for PenaltyBreakBeforeFirstCallParameter, PenaltyBreakFirstLessLess, PenaltyBreakString, PenaltyReturnTypeOnItsOwnLine so the formater go over 80 columns when not doing so would require to screw things up too much.